1/tom-stuart-captions.srt

```diff
@@ -5711,7 +5711,7 @@ to stay in the same order
 1230
 01:16:11,800 --> 01:16:13,740
 in the array, we
-can't use Rubys' sort by.
+can't use Ruby's sort_by.
```

Comment:
I've got back 5 out of 6 sets of corrections, and the consensus seems to be that the captions should say `descendant_count` instead of `'descendant count'` (and `Node` in preferrence over `node`). [This patch](https://github.com/fromPeerToPeer/captions/commit/ed5fd5592cc18c7c906ccc7d7a0d50edd7842f25) changes most of the method names that were quoted to use underscores. I'll update the style guide to reflect this before requesting feedback on the next video.